library/components/SnykPager.vue

```diff
@@ -93,26 +96,36 @@ export default {
       pageNumbersToShow = [];
       let pageToAdd = this.internalPage;
       let numberOfPagesAdded = 0;
-      while (pageToAdd <= this.lastPage && numberOfPagesAdded < 4) {
-        pageNumbersToShow.push(pageToAdd);
-        pageToAdd++;
-        numberOfPagesAdded++;
+
+      // if there are only up to 5 pages, only show the valid pages, do
```

Comment:
Yeah I remember struggling with this once 😅

Reply:
Yeah... It was a teensy bit irritating to realize I'd missed that case earlier 😅 Glad I'm not alone!